ee/be/modules/040-node-manager/openapi/values.yaml

```diff
@@ -341,6 +341,9 @@ properties:
               # This is a copy from NodeGroup object. We trust that hook will not change it.
               type: object
               additionalProperties: true
+            nodeDrainTimeout:
+              type: [integer]
+              additionalProperties: true
```

Comment:
we think that:
1. we need to add `Seconds` prefix for name and limit min and max value for this field

Reply:
Or we need to ge go lang duration string

Reply:
Changed to nodeDrainTimeoutSecond